core/src/main/java/hudson/model/AdministrativeMonitor.java

```diff
@@ -201,13 +205,36 @@ public Permission getRequiredPermission() {
         return Jenkins.ADMINISTER;
```

Comment:
Hesitating to deprecate this method, but it still matches simple use cases so went for a javadoc instead.